### PR TITLE
[SDK-2083] Removed Facebook App Links Code

### PR DESCRIPTION
**Pull request by nsingh-branch**
## Reference
SDK-2083 -- Remove Facebook App Links code

## Summary
Removed all functions, tests, and misc. code related to Facebook App Links.

## Motivation

It appears that the field is no longer needed on the backend, the usage is very low, and the links may not even work on iOS anymore.

## Type Of Change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing Instructions

Make sure the rest of the SDK is working as usual and there were no unexpected dependencies on the FB app links code.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

**Review by echo-branch — APPROVED** [review]

File: BranchSDK.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1635,7 +1622,7 @@
 				5F2211352894A34000C5B190 /* BNCCallbackMap.m in Sources */,
 				5F2210C82894A34000C5B190 /* NSString+Branch.m in Sources */,
 				5F2210DF2894A34000C5B190 /* UIViewController+Branch.m in Sources */,
-				5F2210FE2894A34000C5B190 /* BNCFacebookAppLinks.m in Sources */,
+				5F2211082894A34000C5B190 /* BranchLogoutRequest.m in Sources */,
```
Comment (gdeluna-branch) [inline BranchSDK.xcodeproj/project.pbxproj:1625]: Wasn't `BranchLogoutRequest` removed?

Comment (nsingh-branch) [reply BranchSDK.xcodeproj/project.pbxproj]: Ah yeah, looks like i messed up my merge conflict fix. Just removed that line too.

**Comment by echo-branch** [comment]
@nsingh-branch Seems one unit test is failing. At a glance, I don't see why it would be related to this change.

**Comment by gdeluna-branch** [comment]
> @nsingh-branch Seems one unit test is failing. At a glance, I don't see why it would be related to this change.

I ran into the same error on my PR, might be transient network issues. rerunning resolved mine.

**Pull request merged**